Applications: nrf5340_audio:  Oct 3381 stop using audio ch l and r as indexes (#22879)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/migration/migration_guide_3.1.rst

   * The :ref:`nrf53_audio_app` has been updated to use the :ref:`net_buf_interface` API to handle audio data.
     This change requires you to update your application code to use the new APIs for audio data handling.
     See :ref:`ncs_release_notes_changelog` for more information.
   * The audio devices are now identified by a location bitfield instead of a channel.

>      This requires the location bitfield to be set according to the user's preference.
>      If using the buildprog.py script, this will handle the correct write to UICR.
>      The user only needs to update the locations in the devices.json file.

Reviewer: ```suggestion
     This requires the location bitfield to be set according to your preference.
     If you are using the :file:`buildprog.py` script, it will handle the correct write to UICR.
     You only need to update the locations in the :file:`devices.json` file.
```
